ts/tsconfig.json

```diff
@@ -23,7 +23,7 @@
         "target": "es2019",
         "module": "es2020",
         "lib": [
-            "es2017",
+            "es2020",
```

Comment:
Could we make this es2020.string instead? String.matchAll appears to be supported in Qt 5.14 + iOS 14, but I'm not sure all of es2020 is.

Reply:
Yes, sure we can. I will update it that way.